app/src/test/java/com/timothyolt/evolutionarydesign/ExampleUnitTest.kt

```diff
@@ -1,8 +1,6 @@
 package com.timothyolt.evolutionarydesign
 
-import org.junit.Test
-
-import org.junit.Assert.*
+import kotlin.test.*
```

Comment:
Could probably just delete this file.

Reply:
Without at least one unit test, the gradle task will not run